line_up.c

```diff
@@ -0,0 +1,42 @@
+# include <iostream>
+# include <algorithm>
+# include <vector> 
+# include <functional> 
+# include <numeric>
+
+void print_vector(std::vector<int> const &v) {
+	for (int c: v) { std::cout << c << " "; }
+	
+	std::cout << std::endl;
+}
+
+int main() {
+	std::vector<int> v = {1, 2, 3, 5, 8, 8, 13, 4, 8};
+	
+	int more_than_four = std::count_if(v.begin(), v.end(), [](int &c) -> bool { return c > 4; });
```

Comment:
you dont need to get `c` by reference because it's int :)
and if you already done that, `const reference`